### PR TITLE
Lesson 13

### PR DESCRIPTION
**Pull request by neyoalao**
Please kindly glance through the first draft of lesson 13.
I currently have no idea what to implement for the checkpoint!

### CODE REVIEW

**Comment by angelsvirkov** [comment]
> I currently have no idea what to implement for the checkpoint!

I think we would have a better picture once we have checkpoints 10, 11 and 12. So for now it is not possible I think

**Comment by neyoalao** [comment]
I can't agree more.
Could you have a look at the write-up? I would like to know if there is something I need to remove or add!

**Comment by impactmass** [comment]
The display types (inline/inline-block) together with CSS Floats can help achieve a horizontal layout for the nav links. Maybe that can be one of the exercise(s) and checkpoint here. We can already write about it in the notes to guide the teacher.   
checkpoint_-_CodeSandbox
The `display: inline-block;` is briefly mentioned in Lesson 10, here we can explain it better.

File: lesson13/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+# CSS Display
+
+In this session, students will get an understanding of the different display types that are available in CSS.
+
+## Class Agenda
+
+1. Normal Flow
+1. CSS Display types
+1. CSS Positions
```
Comment (angelsvirkov) [inline lesson13/index.md:13]:
```suggestion
3. CSS Float
```

**Comment by angelsvirkov** [comment]
@neyoalao let's pull out the `nav`  bar out of header and also add it to all pages in this checkpoint

File: lesson13/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- what does block level and inline elements mean?
+- How buttons can be styled
+- How to style borders (border radius, border style e.g dotted....)
+
+ETA: 20 mins
+
+## 2. CSS Display types
+
+In this section, students get understanding of the display types available `block, inline and inline-block`
+
```
Comment (angelsvirkov) [inline lesson13/index.md:28]:
```suggestion
> Disclaimer: Lesson 12 (link to it) already covered the characteristics and differences between inline boxes and block boxes. This sections of the lecture goes into more details about the different display types and  explains how to change the default display type with the css `display` rules.
```

**Comment by impactmass** [comment]
helping here

File: lesson13/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+## 3. Floats
+
+- Float was originally introduced to simple layouts like wrapping text around an image (borrowed from print design).
+- But it became used to achieve more complex layouts
+- Exercises:
+  - We will restructure our HTML a little bit
+    - Pull out the `nav` content out of the header, so it is a direct child of the `body`
+    - Copy and paste the `nav` across the other HTML files
+    - In index.html,  the `header` should now be below the `nav` and should contain then `h1` and `img`
+    - All of this can be seen in Checkpoint 13
```
Comment (impactmass) [inline lesson13/index.md:40]:
```suggestion
    - Check Checkpoint 13 for a example with completed steps
```

File: lesson13/index.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+## 1. Normal Flow
+
+This section will help the students to understand how webpage elements are laid out by default.
+
+- Last class introduced the idea that everything on the page is a box.
+- By default, these boxes can be stacked up on top of one another, or side by side
+- Revision: What does block level and inline elements mean?
+
+
+## 2. The `display` Property
```
Comment (angelsvirkov) [inline lesson13/index.md:21]: Lesson 12 teaches `inline` and `block` . I told the students that they would learn about `inline-block` in Lesson 13, as well. I was under the impression we were about to teach that here. Should we add it `display: inline-block;`

**Review by angelsvirkov — CHANGES_REQUESTED** [review]
It looks good. I think you might have enough time to properly create a nav menu. I have suggested one part that we might also need to teach

File: lesson13/index.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+## 3. Floats
+
+- Float was originally introduced to simple layouts like wrapping text around an image (borrowed from print design).
+- But it became used to achieve more complex layouts
+- Exercises:
+  - We will restructure our HTML a little bit
+    - Pull out the `nav` content out of the header, so it is a direct child of the `body`
+    - Copy and paste the `nav` across the other HTML files
+    - In index.html,  the `header` should now be below the `nav` and should contain then `h1` and `img`
+    - Check Checkpoint 13 for a example with completed steps
```
Comment (impactmass) [inline lesson13/index.md:40]:
```suggestion
    - Check Checkpoint 13 for example with completed steps
```

File: lesson13/index.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+1. The `display` Property
+1. Floats
+
+## 1. Normal Flow
+
+This section will help the students to understand how webpage elements are laid out by default.
+
+- Last class introduced the idea that everything on the page is a box.
+- By default, these boxes can be stacked up on top of one another, or side by side
+- Revision: What does block level and inline elements mean?
```
Comment (impactmass) [inline lesson13/index.md:18]:
```suggestion
- Revision: What does block level and inline elements mean?
- Discuss `display: inline-block`. How is it similar/different to block and inline?
```

File: lesson13/index.md (hunk 4 above)
Comment (impactmass) [reply lesson13/index.md:22]: Good. I've added it above.

**Review by angelsvirkov — APPROVED** [review]
Looks good to me

**Comment by angelsvirkov** [comment]
@impactmass , you can proceed with merging if you are done with it

**Pull request merged**